cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -43,7 +45,12 @@ public int config() throws Exception {
 
     @Command(name = "clone")
     public int cloneCmd(@Parameters(index = "0") String url) {
-
+        try{
+            url = URLDecoder.decode(url,"UTF-8");
```

Comment:
shouldn't this not only be done for the file path ?

if there are spaces in the git clone part it needs to still be encoded

Reply:
Yes true, it should be there in the blob case only. Would move it down to that specific case 👍

Reply:
should also start adding tests for these changes so we don 't break it in future.

Reply:
Yes , will add tests.